DocumentedExamples/NearestNeighbourDistance.ipynb

```diff
@@ -0,0 +1,1730 @@
+{
```

Comment:
<p>Did you mean to write NaN values?</p><p>Also, "values from cumulative"? Is this unfinished sentence?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/258/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>